generators/setup-docker/templates/fabric-docker/commands-generated-node-build.sh.ejs

```diff
@@ -1,8 +1,8 @@
 <% if(chaincode.lang === "node") { -%>
 printHeadline "Building '<%= chaincode.name %>'" "U1F60B"
```

Comment:
tu bez wcięcia?

Reply:
dodałem wcięcie. przez wcześniejsze problemy z formatowaniem to powstało